### ISSUE TITLE
In track - back & forward doesn't work (Kinsky)

### ISSUE THREAD

**Issue by mightyoakbob** [PeteManchester/MediaPlayer#53]
Hi Pete,

Just noticed that using iPad Kinsky, the ability to move backwards and forwards within a track doesn't work. The wheel moves further round but after that the pi will no longer play and the only way to get music out of it is to start the track again. Is this a known issue or do you have any ideas what the problem is?

Thank you for your time.

Cheers,

Bob.

**Comment by PeteManchester** [comment]
Hi Bob,

No it's not something I've heard of before..

Just tested with iPad Kinsky and Windows Kinsky and both work ok on Flac and MP3.

What format of files are you using?

Do you see any errors in the mpd log?

Cheers,

Pete.

**Comment by mightyoakbob** [comment]
Hi Pete,

I've spent a few hours on this trying to work out why mine has this problem and seemingly no one else's does. Eventually I think I found it. It was the buffer settings.

I was using
audio_buffer_size   "16384"
buffer_before_play  "100%"
With those settings the back/forward in track doesn't function.

I've put it back to 2048 and  10% but I'm now expecting the flac erros again. More testing...

Cheers,

Bob.
